library/core/src/array/mod.rs

```diff
@@ -364,3 +364,61 @@ macro_rules! array_impl_default {
 }
 
 array_impl_default! {32, T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T}
+
+#[cfg(not(bootstrap))]
+#[lang = "array"]
+impl<T, const N: usize> [T; N] {
+    /// Returns an array of the same size as `self`, with function `f` applied to each element.
+    /// The closure will be called on elements 0 up to but excluding N.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #![feature(array_map)]
```

Comment:
We don't typically hide the feature gates in doc examples, to not needlessly confuse users.

Reply:
Ah hiding it was also suggested by another reviewer, I can undo it though